posthog/templates/email/batch_export_run_failure.html

```diff
@@ -0,0 +1,31 @@
+{% extends "email/base.html" %} {% load posthog_assets %} {% load posthog_filters %}
+{% block preheader %}If failures keep occurring we will disable this batch export{% endblock %}
+{% block heading %}PostHog batch export {{ name }} has failed{% endblock %}
+{% block section %}
+<p>
+  There's been a fatal error with your batch export {{ name }} at {{ time }}. Due to the nature of the error, it cannot be retried automatically and requires manual intervention.
```

Comment:
We should probably say `UTC` (I think that's what it is, since we're using `last_updated_at`?

Reply:
It is indeed `UTC` :+1:

Reply:
We can add the `%Z` format code and `strftime` will take care of printing `UTC` or whatever the timezone is.